src/common/guac_rect.c

```diff
@@ -80,3 +80,187 @@ void guac_common_rect_constrain(guac_common_rect* rect, const guac_common_rect*
 
 }
 
+int guac_common_rect_adjust(int divisor, guac_common_rect* rect,
```

Comment:
So this function is intended to adjust the bounds of the given rect such that it is aligned to a grid, yet fully contains the original rect?